dojo/pipeline.py

```diff
@@ -88,10 +88,10 @@ def update_product_access(backend, uid, user=None, social=None, *args, **kwargs)
                     product.authorized_users.add(user)
                     product.save()
                 else:
-                    product_member, created = Product_Member.objects.get_or_create(product=product, user=user)
+                    product_member, created = Product_Member.objects.get_or_create(product=product, user=user, role=Role.objects.get(id=Roles.Owner))
                     if created:
```

Comment:
The 'if created:` block can then go as well. The role for a new user has been set in the line before